src/components/thumbnail-maker/PallettePicker.tsx

```
@@ -1,7 +1,28 @@
+        <button
+          key={variant}
+          className={cn(
+            "h-8 w-8 rounded-[4px]",
+            "bg-cover",
+            currentPalette === variant ? "border-[1px] border-white" : ""
+          )}
+          style={{ background: palette.paletteBackground }}
+          onClick={() => onPaletteChange(variant as PaletteVariant)}
+          aria-label={`Select ${variant} palette`}
+          aria-pressed={currentPalette === variant}
+        />
```

Biome

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)